roles/etcd/templates/etcdctl.sh.j2

```diff
@@ -3,19 +3,24 @@
 # command flags are different between the two. Should work on stand
 # alone etcd hosts and master + etcd hosts too because we use the peer keys.
 etcdctl2() {
- cmd="/usr/bin/etcdctl --cert-file {{ etcd_peer_cert_file }} --key-file {{ etcd_peer_key_file }} --ca-file {{ etcd_peer_ca_file }} -C https://`hostname`:2379 ${@}"
+
+ cmd="--cert-file {{ etcd_peer_cert_file }} --key-file {{ etcd_peer_key_file }} --ca-file {{ etcd_peer_ca_file }} -C https://`hostname`:2379 ${@}"
  if [[ -f /usr/local/bin/master-exec ]]; then
-   /usr/local/bin/master-exec etcd etcd /bin/bash -c "$cmd"
+   etcdctl_path="/usr/local/bin/etcdctl"
```

Comment:
This isn't the path for etcd in the rhel image, does quay.io differ? If so that's a challenge.

Reply:
Yeah, that's a path on Origin image at least